cc-ansible

```diff
@@ -325,4 +325,11 @@ if [[ "${POSARGS[*]}" =~ post-deploy ]]; then
 fi
 kolla_args+=("${POSARGS[@]}")
 
+#load tags to skip
+ANSIBLE_SKIP_TAGS="$(grep -v "^#" ./kolla-skip-tags | paste -sd ',')"
+if [[ "${ANSIBLE_SKIP_TAGS}" ]]; then
```

Comment:
Huh, I didn't know it could be that simple. Is there a difference b/w this and `-n` ?

Reply:
With no arguments, `[[ expr ]]` tests for "is null", where `-n` tests for "is length nonzero"
From what I can find: https://unix.stackexchange.com/a/280432 they result in the same behavior, but `-n` is semantically clearer.